Employers.java

```diff
@@ -0,0 +1,20 @@
+public class Employers implements Salary {

```

Comment:
Employers are not Salary, remember the rule "has a", "is a"
meanwhile, the singular noun is better for class name